### PR TITLE
fix: add Pericles001

### LINKED ISSUE #1211: Pericles Adjovi
## Profile Screenshot

pericles001_profile

### PR DESCRIPTION
**Pull request by Pericles001**
resolves #1211 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Pericles001.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/55035365/176567832-8f8412c9-1f05-49cd-bcef-1cde44d0207e.png",
```
Comment (Panquesito7) [inline profiles/Pericles001.json:2]: I recommend more of an image like this. What do you think? :)

image

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/176586265-9f8acc6f-0954-46eb-aa62-ee3a8be7c10a.png",
```



__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by Pradumnasaraf — COMMENTED** [review]
Hey, This PR is open for a while. If you need any help feel free to let us know.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Pericles001** [comment]
Review suggestions has been applied.

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

File: profiles/Pericles001.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/51391473/176586265-9f8acc6f-0954-46eb-aa62-ee3a8be7c10a.png",
+    "issueId": 1211,
+    "name": "Pericles Adjovi",
+    "username": "Pericles001"
+  }
```
Comment (schmelto) [inline profiles/Pericles001.json:6]:
```suggestion
}
```

**Comment by Pericles001** [comment]
@schmelto, your suggestion has been applied.

File: profiles/Pericles001.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/51391473/176586265-9f8acc6f-0954-46eb-aa62-ee3a8be7c10a.png",
+    "issueId": 1211,
+    "name": "Pericles Adjovi",
+    "username": "Pericles001"
```
Comment (schmelto) [inline profiles/Pericles001.json:5]:
```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/176586265-9f8acc6f-0954-46eb-aa62-ee3a8be7c10a.png",
  "issueId": 1211,
  "name": "Pericles Adjovi",
  "username": "Pericles001"
```

**Review by schmelto — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**